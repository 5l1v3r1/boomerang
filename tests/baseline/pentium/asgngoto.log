Unknown library function f_setarg
Unknown library function f_setsig
Unknown library function f_init
Unknown library function s_wsle
Unknown library function do_lio
Unknown library function e_wsle
Unknown library function s_rsle
Unknown library function e_rsle
COMPUTED JUMP at 0x804875e, pDest = r24
Unknown library function __cxa_atexit
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
in proc atexit adding addrExp r28{0} - 4 to local table
in proc atexit adding addrExp r28{0} - 8 to local table
in proc atexit adding addrExp r28{0} - 12 to local table
in proc atexit adding addrExp r28{0} - 12 to local table
in proc atexit adding addrExp r28{0} - 16 to local table
in proc atexit adding addrExp r28{0} - 20 to local table
in proc atexit adding addrExp r28{0} - 24 to local table
finding final parameters for atexit
after removing matching assigns (%pc).
=== about to restart decompilation of MAIN__ because indirect jumps or calls have been analysed

after removing matching assigns (%pc).
in proc MAIN__ adding addrExp r28{0} - 8 to local table
in proc MAIN__ adding addrExp r28{0} - 4 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 16 to local table
in proc MAIN__ adding addrExp r28{0} - 16 to local table
in proc MAIN__ adding addrExp r28{0} - 16 to local table
in proc MAIN__ adding addrExp r28{0} - 16 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
in proc MAIN__ adding addrExp r28{0} - 44 to local table
in proc MAIN__ adding addrExp r28{0} - 48 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 32 to local table
in proc MAIN__ adding addrExp r28{0} - 36 to local table
in proc MAIN__ adding addrExp r28{0} - 40 to local table
TYPE ERROR: Existing type at address 0xffffffffffffffd8 is not structure or array type
in proc MAIN__ adding addrExp r28{0} - 44 to local table
TYPE ERROR: Existing type at address 0xffffffffffffffd4 is not structure or array type
in proc MAIN__ adding addrExp r28{0} - 48 to local table
finding final parameters for MAIN__
potential param r28
found new parameter r28
potential param m[r28{0} - 8]
found new parameter m[r28{0} - 8]
finding final parameters for MAIN__
potential param r28
found new parameter r28
potential param m[r28{0} - 8]
found new parameter m[r28{0} - 8]
after removing matching assigns (t[WILDSTR]).
after removing matching assigns (%pc).
in proc main adding addrExp r28{0} - 40 to local table
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 24 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 28 to local table
in proc main adding addrExp r28{0} - 32 to local table
in proc main adding addrExp r28{0} - 40 to local table
could not set type for unknown parameter expression m[r28{0} + 12]
in proc MAIN__ adding addrExp r28{0} - 8 to local table
finding final parameters for atexit
finding final parameters for MAIN__
potential param r28
found new parameter r28
potential param m[r28{0} - 8]
found new parameter m[r28{0} - 8]
local esp has type void * that doesn't agree with type of define void of a library, why?
local edx has type __size32 * that doesn't agree with type of define __size32 of a library, why?
local esp has type void * that doesn't agree with type of define void of a library, why?
