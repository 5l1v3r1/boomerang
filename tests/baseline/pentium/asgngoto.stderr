decompiling entry point main
rprrr